Arc/Arc.js

```diff
@@ -109,12 +131,12 @@ const ArcBase = kind({
 		width: ({radius}) => ri.scaleToRem(radius * 2)
 	},
 
-	render: ({color, endAngle, onClick, radius, size, startAngle, strokeWidth, ...rest}) => {
+	render: ({children, color, endAngle, onClick, radius, size, startAngle, strokeWidth, svgPointerEvents, svgRef, ...rest}) => {
 		const halfStrokeWidth = strokeWidth / 2;
 		const viewBox = `-${halfStrokeWidth} -${halfStrokeWidth} ${radius * 2}  ${radius * 2}`;
 
 		return (
-			<svg {...rest} viewBox={viewBox} pointerEvents="none">
+			<svg {...rest} viewBox={viewBox} pointerEvents={svgPointerEvents} ref={svgRef}>
```

Comment:
How about `<svg pointerEvents="none" {...rest} viewBox={viewBox} ref={svgRef}>`?
Then we don't have to define `svgPointerEvents` prop, but app developers could override it.
CC: @MikyungKim

Reply:
good idea